types/cesium/index.d.ts

```diff
@@ -3488,6 +3493,12 @@ declare namespace Cesium {
         REPLACE
     }
 
+    enum ColorBlendMode {
+        HIGHLIGHT,
+        MIX,
```

Comment:
Do these have to be in the order they are listed in the source, so that the value lines up? https://github.com/CesiumGS/cesium/blob/1.66/Source/Scene/ColorBlendMode.js#L15